web/api/types/types.go

```diff
@@ -0,0 +1,28 @@
+package types
```

Comment:
nit: `apitypes` instead of `types`; `types` is an overused name and it's not the only package called `types` we import in our code. 

Even though it's redundant in the package path, `apitypes` makes it clearer in code that imports it what kind of types are being used.